addons/mass_mailing/models/mailing.py

```diff
@@ -1169,7 +1169,7 @@ def _process_mass_mailing_queue(self):
                     # send the KPI mail only if it's the first sending
                     'kpi_mail_required': not mass_mailing.sent_date,
                 })
-            self.env['ir.cron']._notify_progress(done=count_done, remaining=count_total - count_done)
+            self.env['ir.cron']._commit_progress(1)
```

Comment:
:socks: Would prefer a _commit_progress(processed=1), as when you just scroll it makes understanding easier (processed and remaining are quite self explanatory hence helps understanding what we do)

Reply:
I will update.